packages/agents/lighthouse/src/tasks/processFromRoot/operations/processFromRoot.ts

```diff
@@ -124,6 +127,39 @@ export const processSingleRootMessage = async (
     });
   }
 
+  if (rootMessage.sentTaskId) {
```

Comment:
This info is not being store in DB.
`rootMessage.sentTaskId`, `rootMessage.sentTimestamp`, and `rootMessage.relayerType` are not upserted into DB here